### PR TITLE
fix / change history list trades order

### PR DESCRIPTION
**Pull request by yzhang1994**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:
- make historical trades in ascending timestamp order.

### CODE REVIEW

File: hummingbot/client/command/list_command.py [hunk 1]
```diff
@@ -77,10 +77,12 @@ def _get_trades_from_session(self,  # type: HummingbotApplication
                         .filter(TradeFill.timestamp >= start_timestamp)
                         .order_by(TradeFill.timestamp.desc()))
         if number_of_rows is None:
-            result: Optional[TradeFill] = query.all()
+            result: Optional[TradeFill] = query.all() or []
```
Comment (martinkou) [inline hummingbot/client/command/list_command.py]: the type should be `Optional[List[TradeFill]]`.

File: hummingbot/client/command/list_command.py [hunk 2]
```diff
@@ -77,10 +77,12 @@ def _get_trades_from_session(self,  # type: HummingbotApplication
...
         else:
-            result: Optional[TradeFill] = query.limit(number_of_rows).all()
-        return result or []
+            result: Optional[TradeFill] = query.limit(number_of_rows).all() or []
```
Comment (martinkou) [inline hummingbot/client/command/list_command.py]: ditto

File: hummingbot/client/command/list_command.py (hunk 1 above)
Comment (yzhang1994) [reply hummingbot/client/command/list_command.py]: done

File: hummingbot/client/command/list_command.py (hunk 2 above)
Comment (yzhang1994) [reply hummingbot/client/command/list_command.py]: done

**Pull request merged**